javascripts/discourse/api-initializers/init-search-banner.js

```diff
@@ -40,7 +40,7 @@ export default apiInitializer("0.8", (api) => {
       }
     },
 
-    mouseDownOutside() {
+    clickOutside() {
```

Comment:
Instead of overriding core, perhaps we might want to add an extension point into core?

Reply:
I agree, but I think this is a task for the next revamp of the header. In the current state, with widgets, we override widget logic in lots of places. This whole component, for example, overrides many internals of the `search-menu` widget. It's very fragile. 

Once we move away from widgets though, we should think carefully about extension points here. @janzenisaac @davidtaylorhq are certainly aware, but this (and similar) theme components will need a big Glimmer rewrite.